bugbug/bugzilla.py

```diff
@@ -82,12 +86,20 @@
 
 INCLUDE_FIELDS = ["_default", "filed_via"]
 
+PRODUCTS_TO_EXCLUDE = [
+    "Calendar",
+    "Chat Core",
+    "Invalid Bugs",
+    "MailNews Core",
+    "Thunderbird",
+]
+
 
 def get_bugs(include_invalid: Optional[bool] = False) -> Iterator[BugDict]:
     yield from (
         bug
         for bug in db.read(BUGS_DB)
-        if include_invalid or bug["product"] != "Invalid Bugs"
+        if include_invalid or bug["product"] not in PRODUCTS_TO_EXCLUDE
     )
```

Comment:
Right now only the spam bug model is passing `include_invalid = True`, but we can't be sure in the future. There might be other models which want to use invalid bugs but no bugs from Thunderbird. I think it'd be better to keep the behavior of `include_invalid` as it is now, and add a new `products` argument that is `list(set(PRODUCTS) - set(MAIL_PRODUCTS))` by default.

Reply:
Thank you @marco-c . I will make the changes and submit them soon.